setup.py

```diff
@@ -1,30 +1,11 @@
 #!/usr/bin/env python
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
+from setuptools import setup
 
 from docrane import __version__ as version
 
-packages = [
-    'docrane'
-]
-
 setup(
-    name='docrane',
```

Comment:
you need to use a workaround for lazy-loading pbr with setuptools: http://bugs.python.org/issue15881#msg170215

impl: https://github.com/openstack/neutron/blob/master/setup.py#L20